src/node_zlib.cc

```diff
@@ -73,7 +73,7 @@ void InitZlib(v8::Local<v8::Object> target);
 class ZCtx : public AsyncWrap {
  public:
   ZCtx(Environment* env, Local<Object> wrap, node_zlib_mode mode)
-      : AsyncWrap(env, wrap, AsyncWrap::PROVIDER_ZLIB),
+      : AsyncWrap(env, wrap, AsyncWrap::PROVIDER_ZCTX),
```

Comment:
If you want to keep this equal to the class names, sure, but I would consider `ZLIB` the more obvious/readable choice (so renaming the class might actually be better?).